ci/runGDPR.yml

```diff
@@ -0,0 +1,37 @@
+resources:
+- name: schedule
+  type: time
+  source:
+    start: 2:00 AM
+    stop: 5:00 AM
+    location:   
+
+jobs:
+- name: delete-gdpr-tickets
+  plan:
+  - get: schedule
+    trigger: true
+  - config:
+      container_limits: {}
+      image_resource:
+        source:
+          repository: "((readonly_private_ecr_repo_url))"
+          tag: concourse-gdpr-tickets-latest
+        type: docker-image
+      platform: linux
+      run:
+        args:
+        - -xce
+        - |
+          cd /usr/src/app
+          bundle install
+          bundle exec ruby /usr/src/app/lib/get-latest-ticket-numbers.rb
+          scripts/delete_latest_tickets.sh
+          bundle exec ruby /usr/src/app/lib/get-latest-ticket-numbers.rb
+
+        path: /bin/bash
+      params:
+        ZENDESK_URL: https://govuk.zendesk.com/api/v2
+        ZENDESK_USER_EMAIL: david.pye@digital.cabinet-office.gov.uk
```

Comment:
Can we replace these values with variables? ZENDESK_URL and ZENDESK_USER_EMAIL are sensitive.

Reply:
Yes @adityapahuja , they will be replaced in the next step. We will replace them with a zendesk token, hopefully this afternoon. I wanted to commit this as a working solution

Reply:
Replaced with concourse stored secrets zendesk-url & zendesk-email